lib/modules/manager/ansible-galaxy/util.ts

```diff
@@ -5,7 +5,7 @@ export const blockLineRegEx = /^\s*((\w+):\s*(.*))$/;
 export const galaxyDepRegex = /[\w-]+\.[\w-]+/;
 export const dependencyRegex = /^dependencies:/;
 export const galaxyRegEx = regEx(
-  /^\s+(?<packageName>[\w.]+):\s*["'](?<version>.+)["']\s*/
+  /^\s+(?<packageName>[\w.]+):\s*["'](([=<>])=)?(?<version>.+)["']\s*/
```

Comment:
i think those specifiers need to be part of the version, so renovate can detect them as version range.

we probably need another versioning here. looks like pep440 could match?

Reply:
or it's more like the npm versioning (partly) 🤔

Reply:
@viceice would it be too aggressive to just place a `defaultVersioning` for ansible-galaxy with pep440 versioning here?

https://github.com/renovatebot/renovate/blob/b6670159c4c6c3e6e1b07829604b2c3301f54162/lib/modules/manager/ansible-galaxy/index.ts#L7-L9

Reply:
It should better be places on the datasource.

https://github.com/renovatebot/renovate/blob/638302e519a3c76c01f185b22a4d05289f054b9f/lib/modules/datasource/datasource.ts#L25

you need to override that like his

https://github.com/renovatebot/renovate/blob/638302e519a3c76c01f185b22a4d05289f054b9f/lib/modules/datasource/docker/index.ts#L379

Reply:
Thanks @viceice I changed that. Sorry for the force-push, I reverted the wrong commit :facepalm: but also it's a fresh start for the PR as none of the original changes were there.

Should there be any tests added for this? From what I can see, no pep440-specific tests are done on the datasources that use them, more on the versioning side.